bukuserver/server.py

```diff
@@ -513,7 +515,8 @@ def shell_context():
         bukudb, 'Bookmarks', page_size=per_page, url_render_mode=url_render_mode))
     admin.add_view(views.TagModelView(
         bukudb, 'Tags', page_size=per_page))
-    admin.add_view(views.StatisticView('Statistic', endpoint='statistic'))
+    admin.add_view(views.StatisticView(
```

Comment:
use kwargs for this

so `views.StatisticView('Statistic, endpoint='statistic, bukudb=bukudb)`

Reply:
OK, but frankly i'd prefer similar usage patterns for `add_view`  for Model and non-Model views. now you have to know whether it's a model view (and have `bukudb` as positional parameter) or an other view (with `bukudb` as named parameter). This IMHO is counter intuitive. 
I'ts not the hill to die on for me, but for my code I'd rather avoid this different usage patterns.